lib/cli/commands/getInfo.ts

```diff
@@ -0,0 +1,10 @@
+import { Arguments } from 'yargs';
+import { callback, loadWalliClient } from '../command';
+import { GetInfoRequest } from '../../proto/wallirpc_pb';
+export const command = 'getinfo';
+
+export const describe = 'get info about btcd';
```

Comment:
Just returning information about BTCD is enough for testing but we should extend this command to show useful data about LND, BTCD and walli-server itself.

Reply:
I think we should first look at issue #43.